site/docs/about/glossary.mdx

```diff
@@ -36,3 +36,10 @@ Salt is the J.P. Morgan design system, an open-source solution for building exce
 Salt is under active development. At this stage, we don't recommend theming unless absolutely necessary.
 
 For an overview of theming, refer to our [theming documentation](/salt/themes/index).
+
+
+## Status visibility
+
+For disabled components, status information is not displayed. This approach is based on accessibility principles, as disabled elements don't require visibility in the UI.
+
+In contrast, read-only components display status information. These elements contain relevant information applicable to the workflow. Displaying status ensures accessibility and communicates necessary information.
```

Comment:
Not entirely sure accessibility fits in here... maybe another one for @jake-costa to comment on.